requirements.txt

```diff
@@ -33,6 +32,7 @@ dynesty
 epsie==0.7.1
 
 # For LDG service access
+ciecplib>=0.4.4
```

Comment:
Ah, this will be pulled in from here for most of the tests. I think if you move this to a requirements-igwn.txt and then install that also for the docker image (so that it gets pulled in for singularity I think by IGWN?), then the tests should mostly pass. I suspect a few other changes may be needed for the docker image, but not globally. 

@spxiwh Am I forgetting anything you all use that will need these?